src/server/mod.rs

```diff
@@ -53,6 +55,11 @@ impl SpaceapiServerBuilder {
         self
     }
 
+    pub fn redis_pool(mut self, redis_pool: r2d2::Pool<r2d2_redis::RedisConnectionManager>) -> Self {
+        self.pool = Some(redis_pool.clone());
```

Comment:
Is the `clone()` needed here? Since we pass by value it shouldn't, right?

Reply:
Oh, right :D